Samples/SPM-Dynamic/Package.swift

```diff
@@ -7,16 +7,16 @@ let package = Package(
     name: "SPM-Dynamic",
     platforms: [.macOS(.v10_13)],
     products: [
-        .library(name: "SPM-Dynamic", type: .dynamic, targets: ["SPM-Dynamic"])
+        .executable(name: "SPM-Dynamic", targets: ["SPM-Dynamic"])
     ],
     dependencies: [
         // branch is replaced in CI to the current sha
-        .package(name: "Sentry", url: "https://github.com/getsentry/sentry-cocoa", .branch("main") )
+        .package(name: "Sentry", path: "../../../sentry-cocoa")
```

Comment:
No. We dont change this anymore, we change the Package.swift that this is targeting to.
During release we wont have the the final URL yet to be able target to that artefact, but we target to the artefact that was saved to be used as the final asset.

Reply:
But then sample tests kind of get obsolete. The purpose of this sample is to include sentry-cocoa via SPM, similar to what our users would. Can't we use `binaryTarget` with the URL pointing to the local built XCFramework? That would be excellent.

Reply:
>But then sample tests kind of get obsolete.

IMO it was always obsolete. If `swift build` works, so will targeting the Package.swift, at least I never saw this tests failing if `swift build` is working.

> Can't we use binaryTarget with the URL pointing to the local built XCFramework? That would be excellent.

This is what the test is doing. The Package.swift file is now fixed to point to the final Package.swift (git URL or path does not matter). Instead of targeting an external ZIP URL, our Package.swift is now targeting a local ZIP file. In the end, I believe this replicates the same experience our users will have